test/sqllogictest/transform/column_knowledge.slt

```diff
@@ -710,12 +705,9 @@ EOF
 
 # Multiple bindings, NOT NULL knowledge
 #
-# This is currently masked by identical work done by "non_nullable" at the
-# moment. I had to swap the order of "non_nullable" and "column_knowledge" to
+# This is currently masked by identical work done by the "non_nullable"
+# transform. I had to swap the order of "non_nullable" and "column_knowledge" to
 # see this transform in action in https://optimizer-trace.dev.materialize.com/.
-#
```

Comment:
Changes between `v1` and `v2` are because we can now handle the constant propagation of `l1.#0 = false`.